### DISCUSSION TITLE
Editing SecurityOnion's HTML

### DISCUSSION THREAD

**Discussion by Sshahar** [Security-Onion-Solutions/securityonion#4507] [Unsupported Versions] [answered]
Is there a simple way to change SO's browser display?
I want to:
1. change the href of TheHive to an external server
2. add tools to the list

**Comment by EchoGangster** [comment, marked as answer]
If you are talking about the main page when you launch the SOC console it is possible.  It is in markdown language and the reference is in the documentation.  My organization add some links to launch other services.  

https://docs.securityonion.net/en/2.3/soc.html?highlight=markdown#security-onion-console-soc

**Reply by Sshahar** [reply]
Thank you very much :)

**Reply by Sshahar** [reply]
for some reason, I don't have the files `tools.json` and `motd.md` under `/opt/so/saltstack/default/salt/soc/files/soc/`. 
Any idea what should I do?

**Reply by dougburks** [reply]
Make sure you're running the latest version:
https://docs.securityonion.net/en/2.3/soup.html

**Reply by facyber** [reply]
Hi @dougburks is it also possible to permanently change logo and that text "Brought by you"? I found how to change it inside the docker container but that is temporarily only, until soc container is restarted.

**Reply by Cra5hedC0w** [reply]
I'd like to know this as well, if it is possible to change the security onion logo.
